src/cachecache.ml

```diff
@@ -1 +1,2 @@
 module Lru = Lru
+module Through = Through
```

Comment:
Now that we have types in `s.ml`, maybe you could constrain the type of `Lru` here to make sure it has the right type? (For `Through` you're already doing it in `though.mli`)